marc_to_solr/lib/traject_config.rb

```diff
@@ -972,6 +972,17 @@
   end
 end
 
+to_field 'lc_facet' do |record, accumulator|
+  if record['050']
+    if record['050']['a']
+      first_letter = record['050']['a'].lstrip.slice(0, 1)
+      letters = /([[:alpha:]])*/.match(record['050']['a'])[0]
+      accumulator << Traject::TranslationMap.new("callnumber_map")[first_letter]
+      accumulator << "#{Traject::TranslationMap.new('callnumber_map')[first_letter]}:#{Traject::TranslationMap.new('callnumber_map')[letters]}"
+    end
+  end
```

Comment:
Could we change this to something like
```suggestion
  if record.dig('050', 'a')
    first_letter = record['050']['a'].lstrip.slice(0, 1)
    letters = /([[:alpha:]])*/.match(record['050']['a'])[0]
    accumulator << Traject::TranslationMap.new("callnumber_map")[first_letter]
    accumulator << "#{Traject::TranslationMap.new('callnumber_map')[first_letter]}:#{Traject::TranslationMap.new('callnumber_map')[letters]}"

end
```